File: recaudacion/2.Codigo/backend/RecaudacionApiReciboIngreso/Domain/FuenteFinanciamiento.cs
namespace RecaudacionApiReciboIngreso.Domain
{
    public class FuenteFinanciamiento
    {
        public int FuenteFinanciamientoId { get; set; }
        public string Codigo { get; set; }
        public string Descripcion { get; set; }
        public string RubroCodigo { get; set; }
        public string RubroDescripcion { get; set; }
    }
}

File: recaudacion/2.Codigo/frontend/src/app/core/models/fuentefinanciamiento.ts
export class FuenteFinanciamiento {
    fuenteFinanciamientoId: number;
    codigo: string;
    descripcion: string;
    rubroCodigo: string;
    rubroDescripcion: string;
    estado: boolean;
    usuarioCreador: string;
    usuarioModificador: string;
    index:number;
}

export class FuenteFinanciamientoFilter {
    pageNumber: number;
    pageSize: number;
    sortColumn:string;
    sortOrder: string;
    fuenteFinanciamientoId: number;
    codigo: string;
    descripcion: string;
    estado: boolean;
}
